open_spiel/games/CMakeLists.txt

```diff
@@ -272,355 +275,120 @@ add_library(bridge_double_dummy_solver OBJECT
-               $<TARGET_OBJECTS:tests>)
-add_test(pathfinding_test pathfinding_test)
-
-add_executable(pentago_test pentago/pentago_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(pentago_test pentago_test)
-
-add_executable(phantom_ttt_test phantom_ttt/phantom_ttt_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(phantom_ttt_test phantom_ttt_test)
-
-add_executable(pig_test pig/pig_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(pig_test pig_test)
-
-add_executable(quoridor_test quoridor/quoridor_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(quoridor_test quoridor_test)
-
-add_executable(rbc_test rbc/rbc_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(rbc_test rbc_test)
-
-add_executable(sheriff_test sheriff/sheriff_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(sheriff_test sheriff_test)
-
-add_executable(skat_test skat/skat_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(skat_test skat_test)
-
-add_executable(solitaire_test solitaire/solitaire_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(solitaire_test solitaire_test)
-
-add_executable(stones_and_gems_test stones_and_gems/stones_and_gems_test.cc
-               ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(stones_and_gems_test stones_and_gems_test)
-
-add_executable(tarok_test tarok/tarok_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(tarok_test tarok_test)
-
-add_executable(tic_tac_toe_test tic_tac_toe/tic_tac_toe_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(tic_tac_toe_test tic_tac_toe_test)
-
-add_executable(laser_tag_test laser_tag/laser_tag_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(laser_tag_test laser_tag_test)
-
-add_executable(tiny_bridge_test tiny_bridge/tiny_bridge_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>
-               $<TARGET_OBJECTS:algorithms>)
-add_test(tiny_bridge_test tiny_bridge_test)
-
-add_executable(tiny_hanabi_test tiny_hanabi/tiny_hanabi_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>
-               $<TARGET_OBJECTS:algorithms>)
-add_test(tiny_hanabi_test tiny_hanabi_test)
-
-add_executable(trade_comm_test trade_comm/trade_comm_test.cc ${OPEN_SPIEL_OBJECTS}
-               $<TARGET_OBJECTS:tests>)
-add_test(trade_comm_test trade_comm_test)
-
-add_executable(ultimate_tic_tac_toe_test ultimate_tic_tac_toe/ultimate_tic_tac_toe_test.cc
-               ${OPEN_SPIEL_OBJECTS} $<TARGET_OBJECTS:tests>)
-add_test(ultimate_tic_tac_toe_test ultimate_tic_tac_toe_test)
+target_link_libraries(games PRIVATE bridge_double_dummy_solver)
+
+macro(add_game_executable)
```

Comment:
much cleaner
such one line
so neat!
wow

:)

Reply:
After this PR lands, I have another speedup in the pipeline which [precompiles](https://cmake.org/cmake/help/latest/command/target_precompile_headers.html) `spiel.h` and friends. This macro makes it easy to add `target_precompile_headers(${_NAME} REUSE_FROM open_spiel_core)` to each target😉